client/state/sites/selectors/get-jetpack-computed-attributes.js

```diff
@@ -16,7 +15,6 @@ export default function getJetpackComputedAttributes( state, siteId ) {
 	return {
 		canAutoupdateFiles: canJetpackSiteAutoUpdateFiles( state, siteId ),
 		canUpdateFiles: canJetpackSiteUpdateFiles( state, siteId ),
-		canManage: canJetpackSiteManage( state, siteId ),
```

Comment:
this one may still be used by `client/my-sites/types/post-type-unsupported/index.jsx`, could you take a look?

Reply:
[That's a false alarm](https://github.com/Automattic/wp-calypso/blob/013d5dde76442d5b798fda96e932ddbca4484134/client/my-sites/types/post-type-unsupported/index.jsx#L71) - it's capability check.